### PR TITLE
feat: Add Open Graph Meta tags

### PR DESCRIPTION
**Pull request by Panquesito7**
# Pull Request

## Description

- Add Open Graph Meta tags.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
       name="og:description"
       content="The homepage for our lovely little community."
     />
```
Comment (nhcarrigan) [inline index.html:11]: This one needs to be removed since you've added one on line 17.

File: index.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
+    <meta name="description" content="We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way." />
+    <meta property="og:image" content="https://user-images.githubusercontent.com/51391473/144103613-229d1856-7815-4c14-b43c-229646bc66fb.png" />
```
Comment (nhcarrigan) [inline index.html:13]:
```suggestion
    <meta property="og:image" content="https://cdn.nhcarrigan.com/content/avatars/nhcommunity.png" />
```

CDN link.

File: index.html [hunk 3]
```diff
@@ -9,6 +9,12 @@
...
+    <meta property="og:url" content="https://community.nhcarrigan.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:locale" content="en_US" />
+    <meta property="og:description" content="We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way." />
```
Comment (nhcarrigan) [inline index.html:17]:
```suggestion
    <meta property="og:description" content="We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way." />
    <meta property="twitter:card" content="summary_large_image" />
    <meta property="twitter:creator" content="@nhcarrigan" />
```
Let's enable the twitter embed too 🙂

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Thank You

Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Pull request merged**